### DISCUSSION TITLE
Does the plugin still gets attention?

### DISCUSSION THREAD

**Discussion by Peskdt** [pmdevita/CreeperHeal2#14] [General]
Do you plan to continue working with this plugin?

**Comment by pmdevita** [comment]
Yes. I don't release frequent updates especially since this is a side of a side of a side project, but the plugin has reached a level of maturity where it should be fine in nearly every explosion, apart from compatibility with other plugins and a few annoying "entity blocks". You'll mostly see updates to provide support for new blocks added to the game and I'll usually take that time to tackle a few of the outstanding edge cases and compatibility problems. 

I haven't worked on this since December but here is what was last being worked on if that interests you.

There was some effort made into bringing compatibility with the Cannons plugin but that plugin does some odd stuff with explosions which makes it difficult to get a read on just by watching changes in the world. I'm probably going to contact the developer for that plugin and see if I can write a hook for Cannons to just tell CreeperHeal2 what blocks are in the explosion.

There was also some work made for the world white/blacklist but it needs to be finalized.

Finally, there was a big effort to get paintings and item frames working but since these are "entity blocks", they are actually "killed" by the explosion before CreeperHeal2 is alerted, making it very difficult to react and save them. Trying to get around this has proven difficult as well and comes with some potential performance costs.
